src/presentation/api/reports/ReportsController.ts

```diff
@@ -0,0 +1,35 @@
+import { getTestTypes, getReports } from '../../../application/service';
+import { Request, Response } from 'express';
+import { Authorized, Get, JsonController, UseBefore, Res, Req, QueryParam } from 'routing-controllers';
+import { hasPermission } from '../../middleware/hasPermission';
+import { VIEW_ADMIN_REPORTS } from '../../../domain/model/authentication/Permissions';
+import { ApiError, apiErrorCodes } from '../../dtos/ApiError';
+import { transformReportTestResultsToReport } from '../../transformers/reports/transformReportTestResultsToReport';
+
+import { writeToStream } from '@fast-csv/format';
+
+@Authorized()
+@JsonController('/v1/reports')
+export class ReportsController {
+  private getReports = getReports;
+  private getTestTypes = getTestTypes;
+
+  // Only used for data reporting/export in MVP. Very heavy query as database grows.
```

Comment:
Is there something we can change in the name of the method to make this comment not needed? 😇